src/io/flutter/preview/PreviewView.java

```diff
@@ -177,14 +180,10 @@ public void onResponse(FlutterOutline widget, JsonObject response) {
     }
 
     @Override
-    public void onFailure(RenderProblemKind kind, @Nullable FlutterOutline widget) {
+    public void onFailure(RenderProblemKind kind, FlutterOutline widget) {
```

Comment:
should widget now be marked `@Nonnull`?

Reply:
No, it is still `@Nullable`.
I restored the annotation.